features/step_definitions/datagovuk_steps.rb

```diff
@@ -1,3 +1,5 @@
+require 'pry'
```

Comment:
I think this was left in from debugging - do we still need this?

Reply:
will remove!